src/code_review/rule.gleam

```diff
@@ -214,3 +213,27 @@ fn accumulate(
     new_context,
   )
 }
+
+// PRETTY PRINTING -------------------------------------------------------------
+
+/// TODO: Just an initial repr for testing, someone good at making things pretty
+///       will need to update this.
+///
+pub fn pretty_print_error(error: Error) -> String {
```

Comment:
**Strongly** recommend you use [hug](https://hexdocs.pm/hug/) for this!

Reply:
Will do! Hug looks great